app/controllers/obituaries_controller.rb

```diff
@@ -17,6 +17,7 @@ def create
   end
 
   def show
+    @user = current_user
```

Comment:
I'm not sure why this was added. Will it break if there isn't a current user?

Reply:
I just needed a way to pass in the user to the view, to make sure whether or not to allow them access to the email button.